### ISSUE TITLE
Command exited with code -1073741502

### ISSUE THREAD

**Issue by GandaG** [appveyor/ci#822]
I'm getting _Command exited with code -1073741502_ on my builds as of today. [This ](https://ci.appveyor.com/project/GandaG/fomod-validator/build/26)was the first one to fail, subsequent ones were me trying to understand what was going on. 

No idea what to do :/

**Comment by FeodorFitsner** [comment]
These are environment updates: http://www.appveyor.com/updates - not sure what might have broken your build. Could it be some external dependency you are downloading to support your build? Have you tried re-running the last successful commit?

**Comment by GandaG** [comment]
I only use miniconda, and it doesn't seem to have changed. Re-running latest successful commit now!

**Comment by GandaG** [comment]
Same code with the last successful commit, [here](https://ci.appveyor.com/project/GandaG/fomod-validator/build/38). Anything I can do?

**Comment by FeodorFitsner** [comment]
OK then, try seeing via RDP what's going on there while the build is running: http://www.appveyor.com/docs/how-to/rdp-to-build-worker. Maybe there is more information in Windows event log.

**Comment by GandaG** [comment]
I haven't been able to use RDP yet but I've narrowed it down to miniconda'a python misbehaving. It is simply not being called or is being silenced, no idea why.

**Comment by FeodorFitsner** [comment]
Btw, there are Minicondas pre-installed on build workers: http://www.appveyor.com/docs/installed-software#python

**Comment by GandaG** [comment]
Yes, I'm using that one, first line of my config file: `set PATH=C:\Miniconda-x64;C:\Miniconda-x64\Scripts;%PATH%`

Running:
```
python --version
pip list
```
outputs:
```
Python 2.7.11 :: Continuum Analytics, Inc.
conda (3.19.0)
conda-env (2.4.5)
menuinst (1.3.1)
pip (7.1.2)
pycosat (0.6.1)
pycrypto (2.6.1)
PyYAML (3.11)
requests (2.9.0)
setuptools (18.8.1)
wheel (0.26.0)
You are using pip version 7.1.2, however version 8.1.2 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.
```
Running the same in conda's environment outputs nothing and later throws that error.

**Comment by GandaG** [comment]
RDP'd into the build at the end, apparently the python executable inside miniconda is unsigned (maybe?). [Screenshot](http://i.imgur.com/fyZX7RF.png).

**Comment by FeodorFitsner** [comment]
Interesting. Do you (re)install Miniconda during the build?

**Comment by GandaG** [comment]
No, I simply add it to the path according to an issue somewhere in here.

`set PATH=C:\Miniconda-x64;C:\Miniconda-x64\Scripts;%PATH%`

**Comment by FeodorFitsner** [comment]
Weird, it was installed from official source/installer.

While being on RDP can you try re-installing it to see if that helps?

**Comment by GandaG** [comment]
It didn't help :/ [Screenshot](http://i.imgur.com/0qpL0ZD.png).

**Comment by FeodorFitsner** [comment]
What version of Miniconda is that?

**Comment by GandaG** [comment]
The one found [here](http://conda.pydata.org/miniconda.html), so I'm guessing 4.1.0, not really sure how to get any other.

**Comment by FeodorFitsner** [comment]
OK, can you check if there is any clue in Windows event log (most probably Application)?

**Comment by GandaG** [comment]
The error for when I tried: http://i.imgur.com/YYYmLnP.png It is similar to the errors reported during the build.
The report: http://i.imgur.com/Lqcp8uT.png

**Comment by FeodorFitsner** [comment]
Yeah, but why its path is `C:\Miniconda-x64\envs\fomod-validator\python.exe`? Should be `C:\Miniconda-x64\python.exe`? I've just tried to run it from original path and it worked.

**Comment by GandaG** [comment]
No, that is the root environment, the point of conda is to be able make your own virtual environments (it defaults under envs/) and install whatever you want in it. It's a package installer and virtual env managar all in one

**Comment by GandaG** [comment]
(The new env is also squeaky clean, while the root already has a lot of things pre-installed, so for a lot of projects it can't be used)

**Comment by FeodorFitsner** [comment]
I'm not an expert in python, but maybe some DLLs are missing in that copied environment?

**Comment by GandaG** [comment]
I don't know a lot about windows, but it works perfectly in Linux :/

In the meantime I also tried with the other miniconda installs (without RDP), no luck.

I'm going to try to install miniconda on my windows machine.

**Comment by GandaG** [comment]
It works [locally](http://i.imgur.com/U36vk66.png).
